arch/xtensa/src/common/xtensa_dumpstate.c

```diff
@@ -288,7 +288,7 @@ void xtensa_dumpstate(void)
   if (CURRENT_REGS)
     {
       memcpy(rtcb->xcp.regs,
-             (uintptr_t *)CURRENT_REGS, XCPTCONTEXT_REGS);
+             (uintptr_t *)CURRENT_REGS, 4 * XCPTCONTEXT_REGS);
```

Comment:
```suggestion
             (uintptr_t *)CURRENT_REGS, XCPTCONTEXT_SIZE);
```
Wouldn't `XCPTCONTEXT_SIZE` be a better choice? It should work the same way and be similar to other architectures.